tests/installation/usb_install.pm

```diff
@@ -0,0 +1,111 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: prepare USB for baremetal installation via ISO.
+# On a minimum system that is launched by ipxe and stops
+# at sshd-server-started, dd installation iso to one usb,
+# write ignition and combustion config to the other usb,
+# and launch the installation with the iso usb.
+# Maintainer: Xiaoli Ai(Alice) <xlai@suse.com>, qe-virt@suse.de
+
+package usb_install;
+use base 'y2_installbase';
+use strict;
+use warnings;
+
+use utils;
+use testapi;
+use bmwqemu;
+use ipmi_backend_utils;
+use Utils::Architectures;
+use LWP::Simple 'head';
+use Utils::Backends 'use_ssh_serial_console';
+
+sub run {
+    assert_screen('sshd-server-started', 5);
+    use_ssh_serial_console;
+    assert_script_run("set -o pipefail");
+
+    # Specify the two usb drives to use
+    my @usb = split('\n', script_output("ls /dev/disk/by-id/ -l | grep -i usb | grep -i -v -E \"generic|part\" | sed 's#^.*\\\/##'"));
+    record_info("Disk info on the machine:", script_output("ls /dev/disk/by-id/ -l; fdisk -l"));
+    die "No proper usb devices!" unless (@usb && scalar(@usb) == 2);
+    my $iso_usb = "/dev/$usb[0]";
+    my $ignition_usb = "/dev/$usb[1]";
+    record_info("Pick usb drive $iso_usb to store ISO and $ignition_usb to store ignition config.");
```

Comment:
medium_usb and provision_usb are better.

Reply:
will do

Reply:
done